redaxo/src/core/lib/console/setup/run.php

```diff
@@ -27,19 +27,19 @@ protected function configure()
     {
         $this
             ->setDescription('Perform redaxo setup')
-            ->addOption('lang', null, InputOption::VALUE_REQUIRED, 'System language e.g. "de_de" or "en_gb"')
+            ->addOption('lang', null, InputOption::VALUE_REQUIRED, 'System language e.g. "de_de" or "en_gb"', null, fn () => rex_i18n::getLocales())
```

Comment:
Ich habe die Zeile doch nochmal geändert, weg von `rex_i18n::getLocales(...)`.
Ich hatte mich gewundert, wieso unten bei timezone `DateTimeZone::listIdentifiers(...)` nicht klappte.
Grund ist: An die Function wird von Symfony ein `CompletionInput`-Objekt übergeben. Das kollidiert mit dem eigentlichen Parametern von `listIdentifiers`.
Bei `rex_i18n::getLocales` klappte es, weil die Methode gar keine Params hat. Aber wenn wir da mal einen Parameter anlegen würden, gäbe es hier dann auch eine Kollision. Daher lieber direkt schon mit einem Function-Wrapper.